frontend-react/src/contexts/Session/SessionProvider.tsx

```diff
@@ -5,6 +5,7 @@ import {
     UserClaims,
 } from "@okta/okta-auth-js";
 import { useOktaAuth } from "@okta/okta-react";
+import axios, { AxiosError } from "axios";
```

Comment:
I understand this _might_ be out of the scope of this PR, but we do use both `fetch` and `axios` in our project. Is it time that we hard cut to one or the other? 

IE in: AdminOrgEdit.tsx AdminLastMileFailuresTable.tsx EditSenderSettings.tsx

Reply:
Yes. Axios is currently the pattern (for context, established even before me), which is why it is continuing to be used. Getting everything in the project moved to using these new hooks will naturally align the project in one direction.